src/clone/Cloner.php

```diff
@@ -0,0 +1,250 @@
+						EE::error( 'Unable to sync certs.' );
+					}
+
+					$ssl_args .= ' --ssl-crt=\'' . '/tmp/' . basename( $assoc_args['ssl-crt'] ) . '\'';
+					$ssl_args .= ' --ssl-key=\'' . '/tmp/' . basename( $assoc_args['ssl-key'] ) . '\'';
+				}
+				if ( $assoc_args['wildcard'] ?? false ) {
+					$ssl_args .= ' --wildcard';
+				}
+			}
+
+			return $ssl_args;
+		}
+
+		if ( ! empty( $site_details['site_ssl'] ) ) {
+			// If name of src and dest site are same
+			if ( $this->name === $source_site->name ) {
+				if ( $site_details['site_ssl'] === 'le' || $site_details['site_ssl'] === 'custom' ) {
+					EE\Site\Cloner\Utils\copy_site_certs( $source_site, $this );
+					$ssl_args .= ' --ssl=custom --ssl-key=\'' . '/tmp/' . $source_site->name . '.key\' --ssl-crt=\'/tmp/' . $source_site->name . '.crt\'';
+					if ( $site_details['site_ssl_wildcard'] ) {
+						$ssl_args .= ' --wildcard';
+					}
+				} elseif ( $site_details['site_ssl'] === 'inherit' ) {
+					EE::warning( 'Unable to enable SSL for ' . $this->name . ' as the source site was created with --ssl=custom. You can enable SSL with \'ee site update\' once site is cloned.' );
+				} elseif ( $site_details['site_ssl'] === 'self' ) {
+					$ssl_args .= ' --ssl=' . $site_details['site_ssl'];
+					if ( $site_details['site_ssl_wildcard'] ) {
+						$ssl_args .= ' --wildcard';
+					}
+				}
+			} else {
+				// If name of src and dest site are note same
+				if ( $site_details['site_ssl'] === 'custom' || $site_details['site_ssl'] === 'inherit' ) {
+					EE::warning( 'Unable to enable SSL for ' . $this->name . ' as the source site was created with --ssl=custom or --ssl=inherited. You can enable SSL with \'ee site update\' once site is cloned.' );
+				} elseif ( $site_details['site_ssl'] === 'self' ) {
+					$ssl_args .= ' --ssl=' . $site_details['site_ssl'];
+					if ( $site_details['site_ssl_wildcard'] ) {
+						$ssl_args .= ' --wildcard';
+					}
+				}
+			}
+		}
+
+		return $ssl_args;
+	}
+
+	private function get_site_create_command( Site $source_site, $assoc_args ): string {
+		$site_details = $source_site->site_details;
+		$command      = 'ee site create ' . $this->name . ' --type=' . $site_details['site_type'];
+
+		if ( in_array( $site_details['site_type'], [ 'html', 'php', 'wp' ] ) ) {
+			if ( '/var/www/htdocs' !== $site_details['site_container_fs_path'] ) {
+				$path    = str_replace( '/var/www/htdocs/', '', $site_details['site_container_fs_path'] );
+				$command .= " --public-dir=$path";
+			}
+			$command .= $this->get_ssl_args( $source_site, $assoc_args );
+		}
+
+		if ( in_array( $site_details['site_type'], [ 'php', 'wp' ] ) ) {
+			if ( ! empty( $site_details['cache_nginx_browser'] ) ) {
+				$command .= " --cache";
+			}
+			if ( ! empty( $site_details['cache_host'] ) ) {
+				$command .= " --with-local-redis";
+			}
+			if ( ( ! empty( $site_details['db_host'] ) && 'php' === $site_details['site_type'] ) || 'wp' === $site_details['site_type'] ) {
```

Comment:
`|| 'wp' === $site_details['site_type']` This part is not needed. After removing this the next `if` condition will also not be needed.